src/components/application_manager/src/display_capabilities_builder.cc

```diff
@@ -102,6 +105,11 @@ DisplayCapabilitiesBuilder::display_capabilities() const {
   return display_capabilities_;
 }
 
+bool DisplayCapabilitiesBuilder::is_window_resumption_needed() const {
+  LOG4CXX_AUTO_TRACE(logger_);
+  return is_window_info_saved;
```

Comment:
@ychernysheva I believe you don't need this variable.
Just check that `window_ids_to_resume_` contains anything with `id != kDefaultWindowID`

Reply:
Fixed in 041b18b68fcb8cae8c7116d2442b340553734285